pkg/services/mssql/dbms_registered_update.go

```diff
@@ -0,0 +1,35 @@
+package mssql
+
+import (
+	"context"
+
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+)
+
+func (d *dbmsRegisteredManager) ValidateUpdatingParameters(
+	service.Instance,
+) error {
+	return nil
+}
+
+func (d *dbmsRegisteredManager) GetUpdater(
+	service.Plan,
+) (service.Updater, error) {
+	return service.NewUpdater(
+		service.NewUpdatingStep("updateAdministrator", d.updateAdministrator),
+		service.NewUpdatingStep("testConnection", d.testConnection),
```

Comment:
Good point and thanks for the suggestion. Will follow it. More further, I think `testConnection` should also be a function call in provisioning step so that it wouldn't look weird.